site/js/video.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function(){
+
+  function isMobile() {
+    return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+  }
+
+  if (!isMobile()) {
+
+    var video = $('#stuff_video')[0];
+
+    // ---HIDE PLAY/PAUSE BUTTON ON LOAD---
```

Comment:
was accessing the index the only way to get this to work? I'm honestly curious, I've heard different people had various problems accessing the video object.